### PR TITLE
Update app's level

### PR DESCRIPTION
**Pull request by yalh76**

### CODE REVIEW

**Review by tituspijean — APPROVED** [review]

File: apps.json [hunk 1]
```diff
@@ -1020,7 +1020,7 @@
     },
     "ffsync": {
         "category": "synchronization",
-        "level": 0,
+        "level": 6,
```
Comment (alexAubin) [inline apps.json:1023]: wait wat

Comment (tituspijean) [reply apps.json:1023]: 🧟

Comment (tituspijean) [reply apps.json:1023]: Ah, @Josue-T wants to keep it alive despite https://github.com/YunoHost-Apps/ffsync_ynh/pull/40

Comment (yalh76) [reply apps.json:1023]: I did worse, by resurrecting helper `ynh_install_python` at https://github.com/YunoHost-Apps/mediadrop_ynh making mediadrop python2.7 working on bullseye

I will try to make coin level7 on bullseye ^^ but introduced a subject for next meeting https://pad.lqdn.fr/p/yunohost-06-09-2022 `what about using to ynh_install_python ?`

**Pull request merged**